src/backend/modules/stock/stock.service.ts

```diff
@@ -1,72 +1,54 @@
 import { Injectable, NotFoundException } from '@nestjs/common';
+import { combineLatest } from 'rxjs';
 
 import { PrismaService } from '../database/prisma.service';
 import { GetStocksQuery } from './dto/get-stocks.query';
 
 @Injectable()
 export class StockService {
-  constructor(private readonly prisma: PrismaService) {}
+  constructor(private readonly prisma: PrismaService) { }
 
   getMany(query: GetStocksQuery) {
     const take = query.take ? query.take : 10;
 
-    return this.prisma.stock.findMany({
+    return this.prisma.company.findMany({
       skip: query.skip,
       take: take,
       orderBy: {
         [query.orderBy ?? 'stockId']: query.orderType ?? 'desc',
       },
-      where: {
-        Company: {
-          name: query.companyName,
-        },
-      },
       select: {
-        stockId: true,
-        Company: {
-          select: {
-            companyId: true,
-            name: true,
-            description: true,
-          },
-        },
-        quantity: true,
-        priceCents: true,
+        companyId: true,
+        name: true,
+        description: true,
```

Comment:
Company model has 3 fields : id, name, description. I don't think it is needed to filter by any, since endpoint above was meant to return list of all stocks/companies.

Reply:
Yeah sure, but what about "Search" functionality?

Reply:
I will let you know that I have functionality in front-end to filter by **companyName**. I also need to have **quantity** and **priceCents** on front end, since I present them.

Reply:
It should be added by now in #64.